### PR TITLE
feat : Implemented Games page according to design

### LINKED ISSUE #6: Design Login page
- Use proper Oauth component for Google sign in (must)

### LINKED ISSUE #8: Make Games Page

### PR DESCRIPTION
**Pull request by siddhantsingh186**
Fixes #8  Make Games Page

### CODE REVIEW

**Comment by siddhantsingh186** [comment]
> @siddhantsingh186 you mentioned wrong issue here ig.

Sorry, my bad! I have updated the PR comment with the correct issue.

**Comment by prathmeshkhandelwal1** [comment]
Can you add proper Screen shot of pages, as we don't have any netlify preview here.

**Comment by siddhantsingh186** [comment]
> Can you add proper Screen shot of pages, as we don't have any netlify preview here.

Sure
games2
games1
games3
.

File: src/components/games/Games.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+function Games() {
+  return(
+      <div className = "games">
+        <div className = "games__head">
+          <p className = "games__head__text">Games</p>
+        </div>
+        <div className  = "games__cards">
+          <div className  = "games__cards__top">
+            <img className = "games__cards__top__csgo" alt = "" src = "images/CSGO.png" ></img>
```
Comment (prathmeshkhandelwal1) [inline src/components/games/Games.js:12]: Formatting error.

File: src/components/games/Games.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            <img className = "games__cards__top__pubg" alt = "" src = "images/PUBG.png" ></img>
+          </div>
+          <div className  = "games__cards__middle">
+            <img className = "games__cards__middle__fifa" alt = "" src = "images/fifa.png" ></img>
+            <img className = "games__cards__middle__apex" alt = "" src = "images/apex.png" ></img>
+          </div>
+          <div className  = "games__cards__bottom">
+            <img className = "games__cards__bottom__assassins" alt = "" src = "images/assasin.png" ></img>
```
Comment (prathmeshkhandelwal1) [inline src/components/games/Games.js:20]: here also

File: src/components/games/Games.js (hunk 1 above)
Comment (prathmeshkhandelwal1) [reply src/components/games/Games.js:12]: Use self closing img tag here.

**Review by prathmeshkhandelwal1 — CHANGES_REQUESTED** [review]

**Comment by siddhantsingh186** [comment]
@prathmeshkhandelwal1 requested changes are done in the latest commit.

**Comment by prathmeshkhandelwal1** [comment]
> @prathmeshkhandelwal1 requested changes are done in the latest commit.

Conflicts are remaining, this arises due the merge of another pr by akshat, can you resolve that please.

**Comment by siddhantsingh186** [comment]
> > @prathmeshkhandelwal1 requested changes are done in the latest commit.
> 
> Conflicts are remaining, this arises due the merge of another pr by akshat, can you resolve that please.

Done👍

**Pull request merged**